lib/skate/detours/missed_stops.ex

```diff
@@ -72,11 +73,11 @@ defmodule Skate.Detours.MissedStops do
           connection_end_segment: Skate.Detours.ShapeSegment.t() | nil
         }
   defp missed_segments({connection_start, connection_end}, segmented_shape) do
-    %{index: start_index} = get_index_by_min_dist(segmented_shape, connection_start)
+    start_index = get_index_by_min_dist(segmented_shape, connection_start)
```

Comment:
nit(non-blocking): would've preferred either a specific commit that describes this change or a seperate PR, because IMO, changing the interface adds noise to the overall diff of this change.

Reply:
Good call!

Lemme see if I can get that out promptly.

Reply:
https://github.com/mbta/skate/pull/2664